src/bootlint.js

```diff
@@ -964,6 +966,86 @@ var LocationIndex = _location.LocationIndex;
             }
         });
     });
+    addLinter("W015", function lintNewBootstrap($, reporter) {
```

Comment:
@cvrebert I've done some refactoring but I'm nervous about breaking out too many small bits. I was wondering if you could take a look at where it is now and let me know if you'd like to see further refactoring for smaller gains or if this is closer to what you are looking for.

Thank you for your time and assistance, I appreciate it.